### PR TITLE
added fun fact for jason-h

### PR DESCRIPTION
**Pull request by JasonHo404**

### CODE REVIEW

**Review by thierrychau — COMMENTED** [review]
No way!!!

File: README.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ Clone this repository (or open in codespace), create a branch with your name, ad
 
 ## 2023-9.1
 | Name | Fun Fact |
-| --- | --- |
```
Comment (Samuel-Lubliner) [inline README.md:34]: dont delete this

**Review by Samuel-Lubliner — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (hibbleD) [reply README.md:34]: No way!

**Review by hibbleD — APPROVED** [review]

**Review by alejlopez139 — APPROVED** [review]

**Review by hibbleD — APPROVED** [review]

**Review by chiucarman — APPROVED** [review]
shark bait muhaha

**Comment by thierrychau** [comment]
that's not right :@

**Pull request merged**